docs/modules/getting_started/examples/code/hdfs.yaml

```diff
@@ -1,27 +1,4 @@
 ---
-apiVersion: zookeeper.stackable.tech/v1alpha1
-kind: ZookeeperCluster
-metadata:
-  name: simple-zk
-spec:
-  version: 3.5.8-stackable0.7.0
-  servers:
-    roleGroups:
-      default:
-        selector:
-          matchLabels:
-            kubernetes.io/os: linux
-        replicas: 3
-        config: {}
----
-apiVersion: zookeeper.stackable.tech/v1alpha1
-kind: ZookeeperZnode
-metadata:
-  name: simple-hdfs-znode
-spec:
-  clusterRef:
-    name: simple-zk
----
 apiVersion: hdfs.stackable.tech/v1alpha1
 kind: HdfsCluster
```

Comment:
I think the resource settings (capacity, memory) can probably be left out? If so I'd omit them to make the example as simple as possible.

Reply:
Done.